spec/index.bs

```diff
@@ -1115,18 +1117,9 @@ To <dfn abstract-op export>get Trusted Types-compliant attribute value</dfn> on
 # Integrations # {#integrations}
 
 <pre class="idl">
-typedef [StringContext=TrustedHTML] DOMString HTMLString;
```

Comment:
Need to remember to update the HTML, Dom Parser, and Sanitiser API specs to use union types instead.

Reply:
As of https://github.com/w3c/DOM-Parsing/pull/82 DOM Parsing spec will no longer contain any TT sinks.